importing-data-5.html

```diff
@@ -86,58 +121,98 @@
 
 </li>
 
-<li><p class="first">Mouse-over the thumbnails of successfully imported images to see a preview.</p>
+<li><p class="first">Import progress will be shown in the <span class="bold">Import # 1</span> tab.</p>
+
+<p>Further data can be queued and imported using the <span class="bold">Select Data to Import</span> tab.</p>
 
-<p>Click on the <span class="bold">View</span> menu to view the image in the <span class="bold">Viewer</span> or <span class="bold">Data Manager</span>, or view the <span class="bold">Import Log</span> and the data <span class="bold">Checksum</span> verification.</p>
+<p>Multiple queues can be run simultaneously.</p>
 
-<p>or close the <span class="bold">Import</span> window to view the data in the <span class="bold">Data Manager</span>.</p>
+<p>A green check is shown in the import tab and in the top right corner of the main Data Manager window when the queue is successfully imported.</p>
+
+<p>Close the <span class="bold">Import</span> window and refresh the data tree to view the data in the <span class="bold">Data Manager</span>.</p>
```

Comment:
Step 6 ends with the instruction `Close the Import window`, but step 7 refers to the Import window again.